CI: ad machine doesn't start correctly

**Describe the bug**
Sometimes, `test` stage in pipeline failed during initial booting of `ad` virtual machine. Issue is random because it could work for CentOS and not for Debian (and vice versa).

Error message:
```
2020-04-21 05:05:03 [ERROR] Error occurred: An error occurred executing a remote WinRM command.

Shell: Cmd
Command: hostname
Message: unknown type: 1235072778
```

I set `VAGRANT_LOG` to `debug` in CI and get two pipelines this night:
- [a passed pipeline for CentOS](https://gitlab.com/inverse-inc/packetfence/-/jobs/519508213)
- [a failed pipeline for Debian](https://gitlab.com/inverse-inc/packetfence/-/jobs/519508227)

--- a/addons/vagrant/Vagrantfile
+++ b/addons/vagrant/Vagrantfile
@@ -16,7 +16,7 @@ Vagrant.configure("2") do |config|
   config.vm.synced_folder ".", "/vagrant", disabled: true
 
   # temp, see https://github.com/hashicorp/vagrant/issues/11539
-  config.winrm.timeout = 120
+  config.winrm.timeout = 600
   
   # use same private key on all machines
   config.ssh.insert_key = false

--- a/addons/vagrant/Vagrantfile
+++ b/addons/vagrant/Vagrantfile
@@ -193,7 +193,6 @@ Vagrant.configure("2") do |config|
                      :libvirt__forward_mode => networks[0]['forward_mode']
 
       srv.vm.provider "libvirt" do |v|
-        v.random_hostname = true
         v.cpus = details['cpus']
         v.memory = details['memory']
       end

